atc/django-atc-demo-ui/atc_demo_ui/templates/atc_demo_ui/base.html

```diff
@@ -10,8 +10,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <!-- React.js -->
-    <script src="{% static 'vendor/react/react-0.12.2.min.js' %}"></script>
-    <script src="{% static 'vendor/react/JSXTransformer-0.12.2.js' %}"></script>
+    <script src="{% static 'vendor/react/react-0.13.3.js' %}"></script>
```

Comment:
I specifically downloaded the non-minified version because the minified one produces incomprehensible stack traces.